src/models/device.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import Column, Integer, Float, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from src.db.db import Base
+
+
+class Device(Base):
+    __tablename__ = "devices"
+
+    # columns
+    id = Column(Integer, primary_key=True, index=True)
+    latitude = Column(Float)
+    longitude = Column(Float)
+    device_name = Column(String(100))
+    created_at = Column(DateTime)
+    updated_at = Column(DateTime)
+    user_id = Column(Integer, ForeignKey("users.id"))
+
+    # relationships
+
+    # owner
+    owner = relationship("User", back_populates="devices")
```

Comment:
Delete this line. use backref in the parent model file.

Reply:
The name `owner` does not match the table name `User`, which seems unnatural.
Perhaps changing the `User` table name to `Owner`?

Reply:
Updated